src/Store/StoreThread.re

```diff
@@ -85,6 +85,7 @@ let start =
       ~minimize,
       ~close,
       ~restore,
+      ~raise,
```

Comment:
`raise` is very ambiguous here. My first instinct, even knowing that you've been working on raising the window, is that refers to raising an exception. And it will in fact also shadow the global `raise` function that does raise exceptions (hence why it's highlighted as a keyword). I would recommend renaming this to `raiseWindow`.

Reply:
Agreed! Not the first time we've had a naming collision with Sdl2 - the `show` function to show a window was easy to confuse for the `show` function to convert to a string. `raiseWindow` sounds good to clarify and avoid the shadowing issue 👍

Reply:
Ah thanks, I totally forgot about the `raise` function shadowing. I'll change that now!